### DISCUSSION TITLE
Hide some card config options unless user has picked "Advanced Mode"

### DISCUSSION THREAD

**Discussion by ZephireNZ** [home-assistant/architecture#571] [General]
In the course of discussing home-assistant/frontend#8804, it was raised that we shouldn't be including every option in the UI editor.

I absolutely agree that too many options in the UI can be overwhelming to new/basic users. However this is counter to the changes within the project to have more and more configurable via the UI, rather than hidden in YAML configs.

To that end, I am suggesting that for some card options, the user must have Show Advanced enabled for it to be visible in the UI. This would keep confusion to a minimum for basic users, while also still giving ability to edit in the UI without switching to YAML mode.

**Comment by jjlawren** [comment]
This is already possible using the `show_advanced_options` attribute set on a flow instance which is tied to a user's browser session.

**Reply by ZephireNZ** [reply]
This isn't regarding flow configuration for integrations, but dashboard cards

**Reply by jjlawren** [reply]
Apologies, didn't read it closely enough. 👍
